src/server/status/index.js

```diff
@@ -29,7 +34,8 @@ export default function (kbnServer, server, config) {
           build_snapshot: matchSnapshot.test(config.get('pkg.version'))
         },
         status: kbnServer.status.toJSON(),
-        metrics: kbnServer.metrics
+        metrics: kbnServer.metrics,
+        stats,
```

Comment:
@jbudz and I discussed this a while back when working on https://github.com/elastic/kibana/pull/10180

Regarding prefixing the endpoints with underscores. While I believe we should align as much as possible with ES, this was done there to differentiate between types/documents as they are also on the root. We don't need to, and shouldn't, prefix our API endpoints.

At the time we decided to keep endpoint at `api/status` in lieu of renaming to stats and we can revisit that if we believe it's beneficial.